README.md

```diff
@@ -1,8 +1,17 @@
-# React + Vite
+# ToDo APP
+## Brief Description
+This repository contains the source code and documentation for the creation of the To-Do List App. This app is designed to help users manage and organize their daily tasks in an efficient and effective way.
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+### how to contribute
+1. Open terminal and clone this repository to your machine local using `git clone repositorylink`.
+2. go to the project directory wtih `cd repo-name`.
+3. Install the dependencies by using Yarn `yarn install
+`.
+4. Run the application with the Yarn command `yarn start`.
 
-Currently, two official plugins are available:
-
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+### Contribution
+- Pahrizal Marup
+- Dharma Chandra Viriya
+- Juhanda Azril Kamal
+- Chandra Kusuma
+- Azis Satya Agni
```

Comment:
I think we don't need this section, since github will automatically track this for us

Reply:
okay sir